README.md

```diff
@@ -69,22 +69,48 @@ docker push ${YOUR-REGISTORY}/openshift-pipelines-operator:${IMAGE-TAG}
 ```
 4. edit the 'image' value in deploy/operator.yaml to match to your image
 
+#### [Running tests](docs/tests.md)
+
 ### Install OLM
 
-**clone OLM repository (into go path)**
+**Clone OLM repository (into go path)**
 
 ```
 git clone git@github.com:operator-framework/operator-lifecycle-manager.git \
-          $GOPATH/github.com/operator-framework/
+          $GOPATH/src/github.com/operator-framework/
+```
+
+**Install OLM**
+
+ensure minikube is installed and docker env is set [see above](#install-minikube)
+
+```
+cd $GOPATH/src/github.com/operator-framework/operator-lifecycle-manager
+```
+```
+GO111MODULE=on NO_MINIKUBE=true make run-local
```

Comment:
I don't see a Makefile in this repo?

Reply:
it is definitely needed. I shall make another PR.

Reply:
The `Makefile` refers to the one in `operator-lifecycle-manager`  see instruction above 
Makefile: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Makefile